vars/distroVersion.groovy

```diff
@@ -13,17 +13,19 @@ String call() {
 
 String call(String distro) {
     String branch = 'master'
-    if (env.BRANCH_NAME.endsWith('2.2')) {
-        branch = 'release/2.2'
+    if (env.BRANCH_NAME.matches('[0-9]+.[0-9]+$')) {
+        branch = env.BRANCH_NAME.replace('.*([0-9]+.[0-9]+)$', 'release/\1')
```

Comment:
Groovy has a matching operator, supports more extended REs, but more importantly, be careful of `.` as it's an _any character_ wildcard:
```suggestion
    if (env.BRANCH_NAME =~ /\d+\.\d+$') {
        branch = env.BRANCH_NAME.replace('.*(\d+\.\d+)$', 'release/\1')
```

But yeah, I now remember why this is here.  :-)  It normalizes other branch names into their landing base for returning the correct element below.